agent/agent.go

```diff
@@ -103,6 +105,9 @@ type Config struct {
 	// TimelineConfig specifies sqlite timeline configuration.
 	TimelineConfig sqlite.Config
 
+	// LocalTimelineConfig specifies local timeline configuration.
+	LocalTimelineConfig sqlite.Config
```

Comment:
How's this different from `TimelineConfig` right above?

Reply:
The local timeline config just has a different sqlite db path. I guess we can just have a single config for both that specifies a directory path and have the overall cluster timeline and local timeline be constant file names in that path.